WTWSMS/localisation/WtWSMS_misc.csv

```diff
@@ -1,99 +1,99 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
```

Comment:
Could you double-check encoding in this file? French localisation appears to be broken.

Reply:
It was indeed broken, now it should be fixed.

Reply:
Perfect, thanks! 👍